src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonic.hpp

```diff
@@ -130,8 +198,9 @@ struct EvolutionMetavars {
   struct ObservationType {};
   using element_observation_type = ObservationType;
 
-  using observed_reduction_data_tags =
-      observers::collect_reduction_data_tags<Event<events>::creatable_classes>;
+  using observed_reduction_data_tags = observers::collect_reduction_data_tags<
+      tmpl::push_back<Event<observation_events>::creatable_classes,
+                      typename Horizon::post_horizon_find_callback>>;
```

Comment:
I think `Horizon::post_interpolation_callback` will work here instead of  `Horizon::post_horizon_find_callback`.  The former is more general and I would prefer it (if it works).

Reply:
Changing this line as you suggest fails to compile with an error "Cannot call apply function of 'observers::ThreadedAtions::WriteReductionData' with DataBoxType.... Full error below.

~~~
Cannot call apply function of 'observers::ThreadedActions::WriteReductionData' with DataBox type 'db::DataBox<brigand::list<observers::Tags::TensorData, observers::Tags::VolumeObserversRegistered, observers::Tags::VolumeObserversContributed, observers::Tags::ReductionObserversRegistered, observers::Tags::ReductionObserversRegisteredNodes, observers::Tags::ReductionObserversContributed, observers::Tags::H5FileLock, observers::Tags::ReductionData<Parallel::ReductionDatum<double, funcl::AssertEqual<funcl::Identity>, funcl::Identity, std::integer_sequence<unsigned long> >, Parallel::ReductionDatum<unsigned long, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Identity, std::integer_sequence<unsigned long> >, Parallel::ReductionDatum<double, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Sqrt<funcl::Divides<funcl::Identity, funcl::Identity> >, std::integer_sequence<unsigned long, 1ul> >, Parallel::ReductionDatum<double, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Sqrt<funcl::Divides<funcl::Identity, funcl::Identity> >, std::integer_sequence<unsigned long, 1ul> >, Parallel::ReductionDatum<double, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Sqrt<funcl::Divides<funcl::Identity, funcl::Identity> >, std::integer_sequence<unsigned long, 1ul> > >, observers::Tags::ReductionDataNames<Parallel::ReductionDatum<double, funcl::AssertEqual<funcl::Identity>, funcl::Identity, std::integer_sequence<unsigned long> >, Parallel::ReductionDatum<unsigned long, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Identity, std::integer_sequence<unsigned long> >, Parallel::ReductionDatum<double, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Sqrt<funcl::Divides<funcl::Identity, funcl::Identity> >, std::integer_sequence<unsigned long, 1ul> >, Parallel::ReductionDatum<double, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Sqrt<funcl::Divides<funcl::Identity, funcl::Identity> >, std::integer_sequence<unsigned long, 1ul> >, Parallel::ReductionDatum<double, funcl::Plus<funcl::Identity, funcl::Identity>, funcl::Sqrt<funcl::Divides<funcl::Identity, funcl::Identity> >, std::integer_sequence<unsigned long, 1ul> > > > >' and arguments 'Parallel::ConstGlobalCache<EvolutionMetavars>, int const&, gsl::not_null<void**>, observers::ObservationId, std::string, std::vector<std::string>, Parallel::ReductionData<Parallel::ReductionDatum<double, funcl::AssertEqual<funcl::Identity>, funcl::Identity, std::integer_sequence<unsigned long> >, Parallel::ReductionDatum<double, funcl::AssertEqual<funcl::Identity>, funcl::Identity, std::integer_sequence<unsigned long> > >'.
If the argument types to the apply function match, then it is possible that the apply function has non-deducible template parameters. This could occur from removing an apply function argument and forgetting to remove its associated template parameters.
~~~

Reply:
Call 12/18/19: @markscheel and @geoffrey4444 agreed to leave this line as-is